configs/retinanet/retinanet_r18_fpn_1x8_1x_coco.py

```diff
@@ -0,0 +1,16 @@
+_base_ = [
+    '../_base_/models/retinanet_r50_fpn.py',
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+
+# data
+data = dict(samples_per_gpu=8)
+
+# optimizer
+model = dict(
+    backbone=dict(
+        depth=18,
+        init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet18')),
+    neck=dict(in_channels=[64, 128, 256, 512]))
+optimizer = dict(type='SGD', lr=0.0025, momentum=0.9, weight_decay=0.0001)
```

Comment:
After discussion, we feel that linear scaling should be given priority, so LR should still be 0.005. But you can add an explanation